### PR TITLE
Exercises Completed (Except Mimsmind)

### PR DESCRIPTION
**Pull request by arnavgoyal17**

### CODE REVIEW

File: mimsmind0.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+""" 
+usage: python mimsmind0.py [length]
```
Comment (danielsgriffin) [inline mimsmind0.py]:
```
Daniels-MacBook-Pro:D08 dsg$ python mimsmind0.py 2
Traceback (most recent call last):
  File "mimsmind0.py", line 98, in <module>
    main()
  File "mimsmind0.py", line 91, in main
    StartGame(numberDigits, numberTries)
TypeError: StartGame() takes 1 positional argument but 2 were given
Daniels-MacBook-Pro:D08 dsg$ 
```

**Comment by danielsgriffin** [comment]
Great job on mimsmind2.

File: mimsmind0.py (hunk 1 above)
Comment (arnavgoyal17) [inline mimsmind0.py]: I just fixed this. You can check again :)

**Comment by danielsgriffin** [comment]
Good job.

**Pull request closed without merging**